### ISSUE TITLE
Iconic Fonts

### ISSUE THREAD

**Issue by elf-pavlik** [apollo-ng/dspace-app-apollong#90]
http://fortawesome.github.io/Font-Awesome/

"Font Awesome gives you scalable vector icons that can instantly be customized — size, color, drop shadow, and anything that can be done with the power of CSS. "

@chron0 how about using it for icons?

**Comment by chron0** [comment]
yeah, I'll close this for now, since it has always been on the roadmap anyways and we have other issues that are far more pressing...

**Comment by elf-pavlik** [comment]
@chron0 how about using milestones for prioritizing? maybe let's talk it over IRC...
i've seen it on Markus' android device today and most icons simply don't show... IMHO one simply can't use the interface this way :(

**Comment by elf-pavlik** [comment]
@chron0 I made a dirty try on it :) looks cool IMO, please check out: https://github.com/apollo-ng/dspace-client/tree/feature/fontawesome
- works fine now on #android devices!

used this one to cherry-pick set of icons for font files http://www.icnfnt.com

**Comment by elf-pavlik** [comment]
another site for custom collection of icons in a font: http://fontello.com
links to all the collections in https://github.com/fontello/fontello

**Comment by elf-pavlik** [comment]
we already use fontello in this rewrite: https://github.com/elevate-festival/neokarto-party
